robotiq_s_model_articulated_gazebo/launch/robotiq_gripper_empty_world.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"/>
```

Comment:
That's right
On Jan 18, 2015 6:02 AM, "G.A. vd. Hoorn" <notifications@github.com> wrote:

> In
> robotiq_s_model_articulated_gazebo/launch/robotiq_gripper_empty_world.launch
> <https://github.com/ros-industrial/robotiq/pull/17#discussion-diff-23131600>
> :
>
> > @@ -0,0 +1,24 @@
> > +<?xml version="1.0"?>
> > +<launch>
> > +  <arg name="limited" default="false"/>
>
> This is probably only for the UR10 xacro you were using?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-industrial/robotiq/pull/17/files#r23131600>.
>